support/mock/mockrt.go

```diff
@@ -371,11 +371,11 @@ type expectedMessage struct {
 }
 
 func (m *expectedMessage) Equal(to addr.Address, method abi.MethodNum, params runtime.CBORMarshaler, value abi.TokenAmount) bool {
-	return m.to == to && m.method == method && m.value == value && reflect.DeepEqual(m.params, params)
+	return m.to == to && m.method == method && m.value.Equals(value) && reflect.DeepEqual(m.params, params)
```

Comment:
comparison using `==` of `abi.TokenAmount` is incorrect, should use `Equal` operator.